Firestore/Source/Model/FSTMutation.mm

```diff
@@ -22,19 +22,28 @@
 #import "Firestore/Source/Model/FSTDocument.h"
 #import "Firestore/Source/Model/FSTDocumentKey.h"
 #import "Firestore/Source/Model/FSTFieldValue.h"
-#import "Firestore/Source/Model/FSTPath.h"
 #import "Firestore/Source/Util/FSTAssert.h"
 #import "Firestore/Source/Util/FSTClasses.h"
 
+#include "Firestore/core/src/firebase/firestore/model/field_path.h"
+
+using firebase::firestore::model::FieldPath;
+
 NS_ASSUME_NONNULL_BEGIN
 
 #pragma mark - FSTFieldMask
 
+@interface FSTFieldMask () {
+  std::vector<FieldPath> _fields;
```

Comment:
C++ fields can just go on the `@implementation`.

Reply:
done